src-vue/src/components/PullRequestsSelector.vue

```diff
@@ -0,0 +1,184 @@
+<template>
+    <div>
+        <el-collapse accordion @change="onChange">
```

Comment:
```suggestion
        <el-collapse @change="onChange">
```

I'd remove the accordion attribute. When I need to look at pull request that needs both Mods and Launcher, it's a bit too much friction to constantly have to open the other collapsed list again ^^

I'm aware with both open the list gets a bit cluttered, hence we probably wanna add a search feature for each repo next that could be used to then filter to the desired PR but that's outside of the scope of this PR :P